Documentation/ApiOverview/SiteHandling/SiteSettingDefinitions.rst

```diff
@@ -8,88 +8,187 @@ Site settings definitions
 =========================
 
 ..  versionadded:: 13.1
+    Site-scoped setting definitions where introduced. They will most likely be
+    the place to configure site-wide configuration, which was previously only
+    possible to modify via Constant Editor, modifying TypoScript constants.
```

Comment:
I don't understand that part "modifying via Constants Editor" and "modifying TypoScript constants"

Reply:
Rephrased it